cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+describe('Article Flow with Pre-set Auth Token', () => {
+  // eslint-disable-next-line max-len
+  const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NjgxNTM0LCJ1c2VybmFtZSI6InRlc3R1c2VyNjAwNiIsImV4cCI6MTczNDI3MTc3MiwiaWF0IjoxNzI5MDg3NzcyfQ.i5DTxajO1KFWmD6UGuHp5hW6P4JhgyVM9OPFNG3d1kE';
 
-  });
+  beforeEach(() => {
+    cy.clearLocalStorage(); cy.clearCookies();
 
-  it('', () => {
+    cy.setCookie('auth', authToken);
 
+    cy.getCookie('auth').should('have.property', 'value', authToken);
   });
 
-  it('', () => {
-
+  it('should create an article successfully', () => {
+    const article = {
+      title: 'Test Article Title',
+      description: 'This is a test article description',
+      body: 'This is the body of the test article',
+      tagList: ['test', 'article']
+    };
+    cy.request({
+      method: 'POST',
+      url: '/api/articles',
+      headers: {
+        Authorization: `Token ${authToken}`
+      },
+      body: {
+        article: {
+          title: article.title,
+          description: article.description,
+          body: article.body,
+          tagList: article.tagList
+        }
+      }
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      const articleSlug = response.body.article.slug;
+      cy.visit('/');
+      cy.contains(article.title).should('be.visible');
```

Comment:
The `cy.contains` command will only verify that the text exists somewhere on the page, but it doesn't guarantee that the text is actually visible to the user. Consider using `cy.get` with a specific selector to ensure the article title is not only present, but also visible in the expected location.